samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/Content/Controls/FlipViewSamplePage.xaml

```diff
@@ -0,0 +1,95 @@
+﻿<Page
```

Comment:
please get the xaml styler extensions for vs and use it to format this file

Reply:
I've it. DO we have a shared configuration that I can use?

Reply:
ill share one in a bit

Reply:
[Exported-2022-10-18.vssettings.txt](https://github.com/unoplatform/uno.toolkit.ui/files/9813628/Exported-2022-10-18.vssettings.txt)
just drop the .txt extensions if you are importing